### DISCUSSION TITLE
"inputs"

### DISCUSSION THREAD

**Discussion by christoph-heinrich** [tomasklaen/uosc#716] [General]
We should talk about this before the release (and before any more translations happen).

What's currently referred to "inputs" is usually referred to as "shortcuts", "key bindings", "hotkeys". "inputs" sounds more like a source of data, like video inputs on a TV or something. mpv documentation consistently calls them "key bindings", so that would be my preferred choice here.

In the german translation I've translated it with that original meaning in mind, where as something more true to "inputs" would have been "Eingaben" in the sense of something the user has entered, or "Eingänge" for something like a signal source.

Looking at the current chinese translation it seems to be a more generic "inputs" as the same symbols are also used for search input "type to search" and "type & ctrl+enter to search", which is also confirmed when translating them back with google translate. Now I don't know any chinese, so maybe those are also commonly used for key bindings as well, but I suspect the choice would have been different if the original was "key bindings" instead.

I should have probably already brought this up in the original PR, but somehow I didn't even think about that.

**Comment by tomasklaen** [comment]
Yeah I agree. I didn't think about it that much when implementing it, just used `inputs` since that's what the config file is called.

I'd prefer it to be a single word. So the choices are I guess:

```
shortcuts
keybinds
```

`keybinds` fits better with mpv documentation, but I get red squiggly lines below it everywhere :) I still favor it more I guess.

**Reply by christoph-heinrich** [reply]
Two words would have an advantage for menu search, because of the initials "kb". Not a big deal, just wanted to mention.

**Reply by tomasklaen** [reply]
I've meant the `uosc/command` name. The text in menu can be "Key bindings".
